CalcPi.java

```diff
@@ -1,6 +1,20 @@
 // Computes an approximation of PI.
 public class CalcPi {
 	public static void main(String [] args) { 
-	    // Replace this comment with your code
+		double approximation = 0;
+		boolean isActionAddition = true;;
+		int currDenominator = 1;
+		
+		for (int i = 0; i < Integer.parseInt(args[0]); i++, currDenominator+=2) {
```

Comment:
1. In condition segment, when you do, i < Integer.parseInt(args[0]), you are calculating Integer.parseInt(args[0]) for every iteration. 
2. In increment, update only the loop variable, update other variables in the end of the loop.  
3. In increment, spacing in the second one..